examples/tools/webpack.config.js

```diff
@@ -3,29 +3,43 @@ import initPlugins from './webpack.plugins';
 
 const distPath = path.resolve(__dirname, '../dist'); // eslint-disable-line no-undef
 const assetsPath = '/assets/js/';
+const isProd = process.env.NODE_ENV === 'production'; // eslint-disable-line no-process-env, no-undef
```

Comment:
You may be able to get rid of these `eslint-disable` comments by adding `/* globals __dirname, process */`